pom.xml

```diff
@@ -106,7 +122,7 @@
              but I justify it by noting that this should be able to be removed when we migrate to
              Storm 1.1.x, which properly supports Kafka 0.10.x.
           -->
-        <global_storm_kafka_version>1.0.1.2.5.0.0-1245</global_storm_kafka_version>
+        <global_storm_kafka_version>1.1.0</global_storm_kafka_version>
```

Comment:
Couldn't the whole block above this be dropped/edited?  It's pure HDP profile now, right?  Or better yet, replaced with an explanation of the new 1.1.0 + Timer explanation.